.github/workflows/sonarcloud.yml

```diff
@@ -38,7 +38,7 @@ jobs:
             -Dsonar.tests=.
             -Dsonar.test.inclusions=**/*_test.go
             -Dsonar.exclusions=**/migrations/**,**/fixtures/**,**/scripts/**,**/vendor/**,**/*_mock.go,internal/common/database/**
-            -Dsonar.coverage.exclusions=**/router/**,**/vendor/**,**/*_mock.go,**/*_test.go,**/cmd/**/*.*
+            -Dsonar.coverage.exclusions=**/router/**,**/vendor/**,**/*_mock.go,**/*_test.go,**/cmd/**/*.*,**/internal/mailer/mailer.go
```

Comment:
why is this excluded?

Reply:
I thought it was considered a schema/model so no need to make the UT

Reply:
the best practice would still be to test the functions related to the schema
imagine you have a payment system to determine the enum string value of a certain type,

```
type PaymentStatus string

const (
     PaymentStatusPending PaymentStatus = "pending"
     PaymentStatusPaid        PaymentStatus = "paid"
)
 
 func (p PaymentStatus) IsActive() {
      if (p == "pending") return false
      return false
 }  
 ```
 
 without a proper unit test in place, this will be incredibly dangerous

Reply:
sure sure